app/retro.css

```
@@ -146,7 +146,7 @@
   .title-bar .title {
     background: #fff;
     cursor: default;
-    font-family: Chicago_12;
+    font-family: CziKareChodz3;
```

Stylelint (17.13.0)

[error] Expected quotes around "CziKareChodz3" (font-family-name-quotes)

(font-family-name-quotes)